src/indicators/drought/ndvi.py

```diff
@@ -85,19 +95,33 @@ def download_ndvi(
             end_date = date.today()
         end_year, end_dekad = _date_to_dekad(end_date)
 
-    dts = [_fp_date(filename.stem) for filename in _RAW_DIR.glob("*.tif")]
+    dts = [
+        _fp_date(filename.stem)
+        for filename in _RAW_DIR.glob(f"{_REGION_ABBR_MAPPING[region]}*.tif")
+    ]
+    for year in range(start_year, end_year):
+        for dekad in range(1, 37):
+            if year == start_year and dekad < start_dekad:
+                continue
+            if year == end_year and dekad > end_dekad:
+                continue
+            if clobber or [year, dekad] not in dts:
+                print(year)
+                _download_ndvi_dekad(region=region, year=year, dekad=dekad)
+
 
-    for year, dekad in zip(range(start_year, end_year), range(1, 37)):
```

Comment:
So sorry @caldwellst I think I told you to make this a zip but was totally wrong, so changed it back

Reply:
Ah woops! I should've tested before changing!